code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,796 @@
+		if(C == M) //the same person can't teleport with itself, it's also so people can't get rid of the "curse" by themselves
+			return
+		do_teleport(M, C, forceMove = TRUE)
+		do_teleport(C, T, forceMove = TRUE)
+		count-- //the second person becomes the first person after running into it because they teleport into the rune, triggering swapping again.
+
+
+
+
+
+
+
+
+/* basically a timestop trap, activate when ANYTHING goes over it, that includes projectiles and what not.*/
+
+
+/obj/item/slimecross/warping/sepia
+	colour = "sepia"
+	runepath = /obj/effect/warped_rune/sepiaspace
+	effect_desc = "Creates a rune that stops time for whatever pass over it."
+	drawing_time = 200 ///much longer to draw than other runes because it fucking stops time
+
+obj/effect/warped_rune/sepiaspace
+	icon_state = "time_space"
+	desc = "The clock is ticking, but in what direction?"
+	var/TS = /obj/effect/timestop///The timestop path here used during the timewarp() proc
+
+obj/effect/warped_rune/sepiaspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/timewarp)
+
+
+///stops time on a single tile, it creates a small runtime common to all timestop.pretty minor compared to the wizard timestop.
+obj/effect/warped_rune/sepiaspace/proc/timewarp()
+	if(locate(TS) in T)//checks if there's already a timestop on the rune. here to avoid the effect triggering multiple time at the same time.
+		return
+	new TS(T, 0, 50) //make a timestop of 0 range so it only affects the tile
+
+
+
+
+
+
+
+/*Cerulean crossbreed : creates a hologram of the last person that stepped on the tile */
+
+/obj/item/slimecross/warping/cerulean
+	effect_desc = "Draw a rune that will make a hologram of the last thing that stepped on it."
+	colour = "cerulean"
+	runepath = /obj/effect/warped_rune/ceruleanspace
+
+/obj/effect/warped_rune/ceruleanspace
+	desc = "A shadow of what once passed these halls, a memory perhaps?"
+	icon_state = "holo_rune"
+	var/obj/effect/overlay/holotile
+
+
+/obj/effect/warped_rune/ceruleanspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/holoform)
+
+///makes a hologram of the mob stepping on the tile, any new person stepping in will replace it with a new hologram
+/obj/effect/warped_rune/ceruleanspace/proc/holoform()
+
+	for(var/mob/living/L in T)
+		if(locate(holotile) in T)//here to avoid multiple person on the rune related bugs
+			qdel(holotile)
+
+		holotile = new(T) //setting up the hologram to look like the person that just stepped in
+		holotile.icon = L.icon
+		holotile.icon_state = L.icon_state
+		holotile.alpha = 100
+		holotile.name = "[L.name] (Hologram)"
+		holotile.add_atom_colour("#77abff", FIXED_COLOUR_PRIORITY)
+		holotile.copy_overlays(L, TRUE)
+
+///destroys the hologram with the rune
+/obj/effect/warped_rune/ceruleanspace/Destroy()
+	..() //needed so the rune still gets stored by the crossbreed
+	qdel(holotile)
+
```

Comment:
this is a cool effect and all, but what about:
alternative effect: after a (not beheaded) corpse or a head remains on this rune for 5 seconds, it will begin to recite everything it said over the course of the round. whether or not this is in reverse order is up to you, Caldony. you can look at the code for the mind reader mutation and for changelings learning their absorb victims' recent spoken phrases.